packages/relative-values/models/src/macrovariables.squiggle

```diff
@@ -114,4 +114,4 @@ vars = {
 } -> Dict.merge(animalModule.asDict -> Dict.mapKeys({|k| "sentientWelfare_" + k}) -> Dict.map({|r| r()}))
 
 fn(a,b) = vars[a] / vars[b]
-vars
\ No newline at end of file
+vars
```

Comment:
This is going to mess with other stuff (some of the plot functionality), please leave out for now

Reply:
@OAGr What do you mean? This patch to me looks like no-op to me, it just adds a trailing new line.

Reply:
I've noticed missing line breaks in your previous PRs, were they intentional?

Reply:
I think this may have been a side-effect of running prettier on the codebase.

Reply:
oh, sorry, I misunderstood this. Was thinking that ``vars`` was being added.